src/pages/Lending/Pool/components/Borrow/Borrow.tsx

```diff
@@ -34,23 +50,49 @@ type BorrowRoutesProps = {
   collateralAssetId: AssetId
   depositAmount: string | null
   onDepositAmountChange: (value: string) => void
+  collateralAccountId: AccountId
+  borrowAccountId: AccountId
+  onCollateralAccountIdChange: (accountId: AccountId) => void
+  onBorrowAccountIdChange: (accountId: AccountId) => void
 }
 
 const BorrowRoutes = memo(
-  ({ collateralAssetId, depositAmount, onDepositAmountChange }: BorrowRoutesProps) => {
+  ({
+    collateralAssetId,
+    depositAmount,
+    onDepositAmountChange,
+    collateralAccountId,
+    borrowAccountId,
+    onCollateralAccountIdChange: handleCollateralAccountIdChange,
+    onBorrowAccountIdChange: handleBorrowAccountIdChange,
+  }: BorrowRoutesProps) => {
     const location = useLocation()
+    const [borrowAsset, setBorrowAsset] = useState<Asset | null>(null)
```

Comment:
nitpick but personally i prefer undefined instead of null unless its for JSX elements